pygeoapi/templates/_base.html

```diff
@@ -103,21 +103,21 @@
     {% block extrafoot %}
     {% endblock %}
     <script>
-      // Requests and embeds JSON-LD representation of current page
-      var xhr = new XMLHttpRequest();
-      var path = window.location.protocol + "//" + window.location.host + window.location.pathname + "?f=jsonld";
-      xhr.open('GET', path);
-      xhr.onload = function() {
-        if (xhr.status === 200) {
-          var head = document.getElementsByTagName('head')[0];
-          var jsonld_datablock = document.createElement('script');
-          jsonld_datablock.type = "application/ld+json";
-          //remove full context path, because search engines don't expect it here, pyld requires it.
-          jsonld_datablock.textContent = xhr.responseText.replace('docs/jsonldcontext.jsonld','');
-          head.appendChild(jsonld_datablock);
-        }
-      };
-      xhr.send();
+      // // Requests and embeds JSON-LD representation of current page
```

Comment:
cc @alpha-beta-soup ^^

Reply:
It's not intended to display anything, this is intended for machine reading of the coverage meta/data.
https://www.w3.org/TR/sdw-bp/#coverages
https://www.w3.org/TR/dwbp/#DescriptiveMetadata

> How to Test
> 
> Check if the metadata for the dataset itself includes the overall features of the dataset in a human-readable format.
> 
> Check if the descriptive metadata is available in a valid machine-readable format.

Commenting it out would break whether search engines can understand the page, but that is fairly hypothetical, since there's no rich results for spatial data (yet?). However there won't ever be rich results for spatial data unless spatial data is published with an eye to concerns like machine readability. That's why the Spatial Data on the Web Best Practices was written and why I made the initial introduction of JSON-LD representations into pygeoapi.

So removing it because it's slow is certainly a breaking change.

A separate HTML template that excludes the JSON-LD request would avoid breaking the behaviour. Or for a particular dataset, an option to disable a JSON-LD representation (maybe rapidly return an HTTP 415 Unsupported Media Type). Perhaps that should be the case for all coverages at least until there's a better way to produce structured metadata for coverages as JSON-LD?

The intention of the code that has been commented out is actually to make that request asynchronously so that initial page load is not affected. Perhaps there's a better way to achieve that.